content/topics/bridging-course-hard-skills/oop-concepts/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Object Oriented Programming concepts
+ready: True
+---
+
+### OOP concepts
+
+One thing a lot of programmers really struggle with is Object Orientated Programming. Once you get your head around it it can be pretty fun. And it is a super powerful tool. Read and understand this
+https://dev.to/charanrajgolla/beginners-guide---object-oriented-programming
```

Comment:
These wont look good.

Also, why not just update https://umuzi-org.github.io/tech-department/topics/java-specific/oop-basics/ with the latest stuff instead of making a whole new page in order to add one new link?

Documentation is like code. It should be cohesive